manifests/server/puppetserver.pp

```diff
@@ -252,22 +252,90 @@
     content => template('puppet/server/puppetserver/conf.d/puppetserver.conf.erb'),
   }
 
-  file { "${server_puppetserver_dir}/conf.d/webserver.conf":
+  file { "${server_puppetserver_dir}/conf.d/auth.conf":
```

Comment:
Semi-related: would it be better to also manage user/group/permissions on these files?